src/Package/Impl/DataInspect/Viewers/ViewerBase.cs

```diff
@@ -3,6 +3,7 @@
 
 using System.Threading.Tasks;
 using Microsoft.R.Components.Extensions;
+using Microsoft.R.DataInspection;
 using Microsoft.R.Debugger;
```

Comment:
Ah, it's because of Microsoft.R.Debugger.Engine. I'll probably rename that to just R.Debugger later on, but I'll remove this (and elsewhere) for now.

---
In reply to: [62136654](https://github.com/Microsoft/RTVS/pull/1731#discussion_r62136654) [](ancestors = 62136654,62136426)